js/hole-ng.js

```diff
@@ -176,25 +175,30 @@ const runCode = document.querySelector('#run a').onclick = async () => {
     status.style.background = data.Pass ? 'var(--green)' : 'var(--red)';
     statusH2.innerText      = data.Pass ? 'Pass 😀'      : 'Fail ☹️';
 
-    result = {
-        arguments: data.Argv.join('\n'),
-        diff:      data.Diff,
-        errors:    data.Err,
-        expected:  data.Exp,
-        output:    data.Out,
-    };
```

Comment:
These only existed because I intended to port the `POST /solutions` route to the API and this time use sane lowercase slightly more verbose names, but the renaming shouldn't be too hard to do if and when I replace the API without needing this mapping.